sympy/geometry/ellipse.py

```diff
@@ -1339,30 +1343,69 @@ class Circle(Ellipse):
     >>> c2.hradius, c2.vradius, c2.radius, c2.center
     (sqrt(2)/2, sqrt(2)/2, sqrt(2)/2, Point2D(1/2, 1/2))
 
+    >>> # a circle object returned from given equation and optional 'x' and 'y' parameters
```

Comment:
Please make this a sentence instead of an inline comment. It would be good to update the top of the docstring to say that input also includes an equation.

Reply:
@smichr  Done. Please take a look